content/en/blog/_posts/2021-12-15-pod-security-admission-beta.md

```diff
@@ -0,0 +1,635 @@
+
+```shell
+kubectl create namespace verify-pod-security
+```
+
+### Deploy demo workloads
+
+Each workload represents a higher level of security that would not pass the profile that comes after it.
+
+For the following examples, use the `k8s.gcr.io/pause` container which is a small container that runs a never ending `sleep` command. Pod Security is not interested in the container, but rather the Kubernetes manifests that are being admitted.
+
+**Privileged level and workload**
+
+For the privileged pod, allow [privilege escalation](/docs/concepts/security/pod-security-standards/#privileged). This allows the process inside a container to gain new processes and can be dangerous if untrusted.
+
+First, let's apply a restricted Pod Security level for a test.
+
+```shell
+# enforces a "restricted" security policy and audits on restricted
+kubectl label --overwrite ns verify-pod-security \
+  pod-security.kubernetes.io/enforce=restricted \
+  pod-security.kubernetes.io/audit=restricted
+```
+
+Next, try to deploy a provileged workload in the namespace.
+
+```shell
+cat <<EOF | kubectl -n verify-pod-security apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pause-privileged
+spec:
+  containers:
+  - name: pause
+    image: k8s.gcr.io/pause
+    securityContext:
+      allowPrivilegeEscalation: true
+EOF
+```
+
+The output is similar to this:
+
+```
+Error from server (Forbidden): error when creating "STDIN": pods "pause-privileged" is forbidden: violates PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container "pause" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "pause" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "pause" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "pause" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
+```
+
+Now let's apply the privileged Pod Security level and try again.
+
+```shell
+# enforces a "privileged" security policy and warns / audits on baseline
+kubectl label --overwrite ns verify-pod-security \
+  pod-security.kubernetes.io/enforce=privileged \
+  pod-security.kubernetes.io/warn=baseline \
+  pod-security.kubernetes.io/audit=baseline
+```
+
+```shell
+cat <<EOF | kubectl -n verify-pod-security apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pause-privileged
+spec:
+  containers:
+  - name: pause
+    image: k8s.gcr.io/pause
+    securityContext:
+      allowPrivilegeEscalation: true
+EOF
+```
+
+The output is similar to this:
+
+```
+pod/pause-privileged created
+```
+
+We can run `kubectl -n verify-pod-security get pods` to verify it is running.
+
+**Baseline level and workload**
+
+The [baseline](/docs/concepts/security/pod-security-standards/#baseline) pod demonstrates sensible defaults but does not allow privilege escalation, so the pod above would fail here too. Baseline accepts null / no values for unset keys.
```

Comment:
```suggestion
The [baseline policy](/docs/concepts/security/pod-security-standards/#baseline) demonstrates sensible defaults but does not allow privilege escalation, so the pod above would fail here too. Baseline accepts null / no values for unset keys.
```